scripts/prison-register.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
```

Comment:
Work in progress. Supports getting all prisons or single prison. Other commands to be tested.
Not sure what to do about the auth bit. Currently uses the licences ADMIN_API_CLIENT_SECRET